docs/concepts/operator.md

```diff
@@ -256,6 +256,29 @@ This range is used by the API server to compute the cluster IPs of `Service`s.
 
 The controller maintains the `Reconciled` condition which indicates the status of an operation.
 
+#### [`GardenCare` Controller](../../pkg/operator/controller/garden/care)
+
+This reconciler performs four "care" actions related to `Garden`s.
+
+It maintains four conditions and performs the following checks:
+
+- `SystemComponentsHealthy`: The conditions of the `ManagedResource`s applied to the runtime cluster are checked (e.g., `ResourcesApplied`).
+- `VirtualGardenComponentsHealthy`: The conditions of the `ManagedResource`s applied to the virtual cluster are checked (e.g., `ResourcesApplied`).
+- `VirtualGardenAPIServerAvailable`: The `/healthz` endpoint of the garden's `virtual-garden-kube-apiserver` is called and considered healthy when it responds with `200 OK`.
+- `VirtualGardenControlPlaneHealthy`: The control plane is considered healthy when the respective `Deployment`s (for example `virtual-garden-kube-apiserver`,`virtual-garden-kube-controller-manager`), and `Etcd`s (for example `virtual-garden-etcd-main`) exist and are healthy.
```

Comment:
```suggestion
```

We thinks it's rather better to drop this condition and move the checks to `VirtualComponentsHealthy`. WDYT?
You probably wanted to align to what we already do for shoots but there we have actual workload in the cluster that we need to check, in contrast to the virtual garden cluster where we'll never have any workload. Hence, checking the `ManagedResource`s and putting the result into a dedicated condition doesn't make much sense.